pkg/api/validation/validation.go

```diff
@@ -1975,6 +1975,9 @@ func validateContainers(containers []api.Container, volumes sets.String, fldPath
 		if len(ctr.Image) == 0 {
 			allErrs = append(allErrs, field.Required(idxPath.Child("image"), ""))
 		}
+		if len(strings.TrimSpace(ctr.Image)) != len(ctr.Image) {
```

Comment:
i don't see how image is special.  we validate every other aspect of the pod spec when doing a validate pod template (volumes, affinity, etc.)